tests/test_complete_run.py

```diff
@@ -0,0 +1,111 @@
+from acme_diags.parameter.core_parameter import CoreParameter
+from acme_diags.parameter.area_mean_time_series_parameter import AreaMeanTimeSeriesParameter
+from acme_diags.parameter.enso_diags_parameter import EnsoDiagsParameter
+from acme_diags.parameter.qbo_parameter import QboParameter
+
+# Due to the large amount of data required to run, this test will be run manually on Anvil
+# (rather than as part of the CI tests).
+
+class TestCompleteRun(unittest.TestCase):
+    def test_complete_run(self):
+        # Anvil
+        # Run `source /lcrc/soft/climate/e3sm-unified/load_latest_e3sm_unified.sh`
+        test_data_prefix = '/lcrc/group/e3sm/public_html/e3sm_diags_test_data'
+        ref_data_prefix = '/lcrc/group/e3sm/public_html/diagnostics/observations/Atm'
+        html_prefix = '.'
+
+        param = CoreParameter()
+
+        param.reference_data_path = os.path.join(ref_data_prefix, 'climatology')
+        param.test_data_path = os.path.join(test_data_prefix, 'climatology/')
+        param.test_name = '20161118.beta0.FC5COSP.ne30_ne30.edison'
+        param.seasons = ["ANN","JJA"]  # Default setting: seasons = ["ANN", "DJF", "MAM", "JJA", "SON"]
+
+        param.results_dir = os.path.join(html_prefix, 'tutorial_2020_all_sets')
+        param.multiprocessing = True
+        param.num_workers = 30
+
+        # Additional parameters:
+        #param.short_test_name = 'beta0.FC5COSP.ne30'
+        #param.run_type = 'model_vs_model'
+        #param.diff_title = 'Difference'
+        #param.output_format = ['png']
+        #param.output_format_subplot = ['pdf']
+        #param.save_netcdf = True
+
+
+        # Set specific parameters for new sets
+        enso_param = EnsoDiagsParameter()
+        enso_param.reference_data_path = os.path.join(ref_data_prefix, 'time-series/')
+        enso_param.test_data_path = os.path.join(test_data_prefix, 'time-series/E3SM_v1/')
+        enso_param.test_name = 'e3sm_v1'
+        enso_param.start_yr = '1990'
+        enso_param.end_yr = '1999'
+
+        qbo_param = QboParameter()
+        qbo_param.reference_data_path = os.path.join(ref_data_prefix, 'time-series/')
+        qbo_param.test_data_path = os.path.join(test_data_prefix, 'time-series/E3SM_v1/')
+        qbo_param.test_name = 'e3sm_v1'
+        qbo_param.start_yr = '1990'
+        qbo_param.end_yr = '1999'
+
+        ts_param = AreaMeanTimeSeriesParameter()
+        ts_param.reference_data_path = os.path.join(ref_data_prefix, 'time-series/')
+        ts_param.test_data_path = os.path.join(test_data_prefix, 'time-series/E3SM_v1/')
+        ts_param.test_name = 'e3sm_v1'
+        ts_param.start_yr = '1990'
+        ts_param.end_yr = '1999'
+
+        runner.sets_to_run = ['lat_lon','zonal_mean_xy', 'zonal_mean_2d', 'polar', 'cosp_histogram', 'meridional_mean_2d', 'enso_diags', 'qbo','area_mean_time_series']
+        runner.run_diags([param, enso_param, qbo_param, ts_param])
+
+        actual_images_dir = './tutorial_2020_all_sets'
```

Comment:
Use `actual_images_dir = param.results_dir`, so that the test can be run outside the directory?